kibot/globals.py

```diff
@@ -82,15 +82,15 @@ def __init__(self):
                 KiCad 6: you should set this in the Board Setup -> Board Finish -> Plated board edge """
             self.copper_finish = None
             """ {pcb_finish} """
-            self.copper_thickness = 35
+            self.copper_thickness:str = "35"
```

Comment:
This introduces an incompatibility.

Reply:
It's a string here too: https://github.com/INTI-CMNB/KiBot/blob/master/kibot/globals.py#L136

Reply:
Thanks, 17911fe1361b2adcba5e32b909ddc7f5c1d151f7 fixes it

Reply:
Good - found because of typing checks ;-).